README.md

```diff
@@ -105,6 +105,19 @@ Bevy is free and open source! All code in this repository is dual-licensed under
 
 at your option. This means you can select the license you prefer! This dual-licensing approach is the de-facto standard in the Rust ecosystem and there are [very good reasons](https://github.com/bevyengine/bevy/issues/2373) to include both.
```

Comment:
```suggestion
at your option, except as otherwise noted (below and/or in individual files).
```
(I think we should probably use the exact wording as in https://github.com/rust-lang/rust/blob/main/COPYRIGHT#L16-L19, except GitHub doesn't let me suggest that far up the document)

That is, this note should be added at the start of this section